bindings/cs/rl.net/LiveModel.cs

```diff
@@ -19,7 +21,7 @@ private static New<LiveModel> BindConstructorArguments(Configuration config)
         }
 
         [DllImport("rl.net.native.dll")]
-        private static extern int LiveModelInit(IntPtr liveModel, IntPtr apiStatus);
+        private static extern int LiveModelInit(IntPtr liveModel, IntPtr apiStatus, trace_logger_callback_t traceLogger);
```

Comment:
Instead of passing in logger this way, it is better to follow the factory pattern and let client lib create the logger based on config.

Reply:
At this place, trace_logger_callback_t traceLogger is passed from C# binding, e.g. it is the callback on C# to trigger Geneva logging. And this is the C# binding interface.  The goal of this PR is to enable this call back.

Reply:
With that said, I suggest we model this callback like an event on the C# side. Take a look at the BackgroundError mechanism.